Week3/homework/exercise4.js

```diff
@@ -0,0 +1,20 @@
+"use strict"
+/*the grocery list should contain bananas and milk at first and then we should be adding other items*/
+let groceryItem = ["bananas", "milk"]
+let i = 0;
```

Comment:
Don't think you need this `i` variable, right?

Reply:
I didn't, already took it off.